topics.md

```diff
@@ -5,17 +5,17 @@ Interested in speaking at an upcoming TechTalk DC event? Add your information he
 This is meant to be a guide -- don't let it overwhelm you. Give us what you've got and we'll get the rest before we need it.
 
 
- * **Session Title**:  
- * **Preferred Timeframe**:  
- * **Event Type**: [Pick one: 50-min Lecture, Lightning Talk, 2-3 Hour Workshop, All Day Workshop (plan for a Saturday)]
- * **Summary**:
- * **Presenter Name(s)**:
- * **Presenter E-mail(s)**:
- * **Presenter GitHub Account**:
- * **Presenter Blog**:
- * **Audience Skill Level**: [All Skill Levels, Beginners, Intermediate, Advanced]
- * **Requirements**: [A/V, room setup, or any other needs]
- * **Code of Conduct**: I have read, understand, and agree to follow the Tech Talk DC Code of Conduct.
+ * **Session Title**: Here's What's Trending In Software Engineering
+ * **Preferred Timeframe**: Any weeknight
+ * **Event Type**: Lightning Talk (~30 minutes with questions)
```

Comment:
When I gave the talk originally at the conference, it was in the 30-minute frame because that's what the organizers wanted, but I can easily extend it to 40-50 minutes as @PatLeong suggests. There is plenty more to address on each slide, and I could even add one more topic that got "honorable mention" the first time around.

By the way, at the end of the talk once the audience is done with questions, I like to turn it around and ask the audience which trends in software engineering they see that I missed. I would be happy for @Wyntuition or anyone else to offer their thoughts if time allows.

I have attached the original presentation.
[trending.pdf](https://github.com/techtalkdc/Open-Governance/files/1116397/trending.pdf)

Just keep in mind that some of the slides--like the one with Oprah (yes, Oprah)--may not make a lot of sense without the context I speak to. Please let me know if you would like clarification.